src/components/FeaturePanel/Climbing/YellowedBadge.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import CheckIcon from '@material-ui/icons/Check';
+import { useClimbingContext } from './contexts/ClimbingContext';
+import { isAscent } from './utils/ascents';
+
+const Text = styled.div``;
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  border-radius: 22px;
+  align-items: center;
+  padding: 2px 8px;
+  background-color: ${({ theme }) => theme.palette.climbing.ascent};
+  font-size: 13px;
+  font-weight: 900;
+  gap: 4px;
+`;
+
+export const YellowedBadge = () => {
+  const { routes } = useClimbingContext();
+  const isVisible = routes.every((route) => {
```

Comment:
Can you please add `routes.length>0 &&` to this line? It shows the badge on empty crags 😅 

eg. https://openclimbing.org/relation/14297819